docs/data/date-pickers/custom-field/BrowserV6SingleInputRangeField.tsx

```diff
@@ -103,9 +119,11 @@ const BrowserSingleInputDateRangeField = React.forwardRef(
       ),
     };
 
-    const fieldResponse = useSingleInputDateRangeField<Dayjs, typeof textFieldProps>(
-      textFieldProps,
-    );
+    const fieldResponse = useSingleInputDateRangeField<
+      Dayjs,
+      false,
+      typeof textFieldProps
+    >({ ...textFieldProps, enableAccessibleFieldDOMStructure: false });
```

Comment:
Yes, but that way people that created a custom UI using an `<input />` won't have anything broken when we switch the default value.

My approach here was to say that they know they want an input since they are handling the DOM, unlike the built-in fields which are able to switch automatically if tomorrow we change the default value.

Reply:
Great point, I didn't think about it. 🙈 
Makes sense to keep it. 👍